tests/tests_fabric/loggers/test_tensorboard.py

```diff
@@ -0,0 +1,229 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import os
+from argparse import Namespace
+from unittest import mock
+from unittest.mock import Mock
+
+import numpy as np
+import pytest
+import torch
+from tests_fabric.test_fabric import BoringModel
+
+from lightning_fabric.loggers import TensorBoardLogger
+from lightning_fabric.loggers.tensorboard import _TENSORBOARD_AVAILABLE
```

Comment:
Do you plan to remove any TB/logger tests from PL?

Reply:
I think it is too soon now. The loggers in Fabric may change in ways that require us to do overrides again on the PL side (yes I am still against sharing the code through the subclass)